docset/winserver2022-ps/international/Copy-UserInternationalSettingsToSystem.md

```diff
@@ -0,0 +1,92 @@
+---
+ms.mktglfcycl: manage
+ms.sitesec: library
+ms.author: ertorres
+author: erik0686
+description: Use this topic to help manage Windows and Windows Server technologies with Windows PowerShell.
+external help file: Microsoft.InternationalSettings.Commands.dll-Help.xml
+keywords: powershell, cmdlet
+manager: aandrejs
+ms.date: 09/08/2021
+ms.prod: w11
+ms.technology:
+ms.topic: reference
+online version: 
+schema: 2.1.0
+title: Copy-UserInternationalSettingsToSystem
+ms.reviewer:
+ms.assetid: 
+---
+
+# Copy-UserInternationalSettingsToSystem
+
+## SYNOPSIS
+Copies the current user's international settings (Windows Display language, Input language, Regional Format/locale and Location/GeoID) to one or both of the following:
+* Welcome screen and system accounts
+* New user accounts
+
+This is a system setting. It can only be changed by a user who has Administrator permissions. Changes take effect after the computer is restarted.
```

Comment:
```suggestion
This is a system setting. It can only be changed by a user who has Administrator permissions. Changes take effect after the device is restarted.
```